File: integrations/php/oicana-php/src/Manifest/PdfExportConfig.php
<?php

declare(strict_types=1);

namespace Oicana\Manifest;

/**
 * How documents are exported to PDF.
 */
final readonly class PdfExportConfig
{
    /**
     * @param list<string> $standards PDF standards the export conforms to, for example `a-3b`
     * @param bool $tagged Whether the PDF is tagged for accessibility
     */
    public function __construct(
        public array $standards,
        public bool $tagged
    ) {
    }
}

File: integrations/java/oicana/src/main/java/com/oicana/manifest/PdfExportConfig.java
package com.oicana.manifest;

import java.util.List;

/**
 * How documents are exported to PDF.
 *
 * @param standards PDF standards the export conforms to, for example {@code a-3b}
 * @param tagged Whether the PDF is tagged for accessibility
 */
public record PdfExportConfig(List<String> standards, boolean tagged) {}
